sdk/storage/azure-storage-file-share/tests/test_file.py

```diff
@@ -1963,11 +2243,11 @@ def test_create_file_with_md5_small(self, storage_account_name, storage_account_
 
         # Assert
 
+    @pytest.mark.live_test_only
```

Comment:
Hmm I don't really love how so many of these tests are Live only just because they are parallel. I think we are losing some testing of other useful things just because all these tests are parallel. I wonder if some of the tests should be made non-parallel so they can be recorded...